chaos_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+function rollback() {
+    git checkout master
+    git checkout HEAD^
+}
+
+/root/.virtualenvs/chaos/bin/python chaos.py || rollback
```

Comment:
Yeah, I thought about this a bit because I've accidentally done it to myself a few times :stuck_out_tongue: 

> Maybe revert up to and including the last chaosbot commit

That sounds like a viable solution